static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -0,0 +1,63 @@
+define([
+    'common/utils/$',
+    'common/utils/template',
+    'text!common/views/giraffe-message.html',
+    'common/views/svgs'
+], function ($,template,giraffeMessage, svgs) {
+    return function () {
+
+        this.id = 'Giraffe';
+        this.start = '2016-07-03';
+        this.expiry = '2016-08-13';
+        this.author = 'Alex Ware';
+        this.description = 'Add a button allowing readers to contribute money.';
+        this.showForSensitive = false;
+        this.audience = 0.001;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Determine the best message for driving contributions.';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+        this.canRun = function () {
+             return window.guardian.config.page.edition == 'UK';
+        };
+
+        var writer = function (linkText, linkHref, copy) {
+            var $newThing = $.create(template(giraffeMessage, {
+                linkText:linkText,
+                linkName: 'idk',
+                linkHref: linkHref,
+                copy: copy,
+                svg: svgs('arrowWhiteRight', ['button--giraffe__icon'])
+            }));
+            var a = $('.submeta');
+            $newThing.insertBefore(a[0]);
+        };
+
+        this.variants = [
+            {
+                id: 'everyone',
+                test: function () {
+                    writer('If everyone were to chip in, the Guardian\'s future would be more secure.', 'https://membership.theguardian.com/contribute?INTCMP=article-1-everyone', 'Please support the Guardian and independent journalism');
```

Comment:
it may be worth talking to @desbo about setting the `success` callback on this, I think the team working on the ab test framework would like to trial their new system out ?